issueWork/JS1__issue/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script defer>
+    const numArr = [15,52,33,24,75];
+    let counter = 0;
+    for(let i = 0; i < numArr.length; i++) {
+      numArr[i]+=counter;
+      counter++;
+    }
+    function consoleLoggingAnArray (arr) {
```

Comment:
Suggestion:

Leverage spacing to improve clarity of your code. I would add a space between your for loop and function declaration. 

In addition to the space, as you progress as a developer, always consider adding a comments to your function declarations and any potentially "confusing" portions of the code. Your fellow developers will thank you!